front/src/modules/pathfinding/components/Itinerary/DisplayItinerary/v2/DisplayItineraryV2.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import type { Position } from 'geojson';
+
+import DestinationV2 from './DestinationV2';
+import OriginV2 from './OriginV2';
+import ViasV2 from './ViasV2';
+
+type DisplayItineraryProps = {
+  zoomToFeaturePoint: (lngLat?: Position, id?: string) => void;
+};
+
+export default function DisplayItinerary(props: DisplayItineraryProps) {
+  const { zoomToFeaturePoint } = props;
```

Comment:
Use const and destructure the props
```suggestion
const DisplayItinerary = ({ zoomToFeaturePoint }: DisplayItineraryProps) => (
```

Reply:
Done